src/components/NcAppNavigation/NcAppNavigation.vue

```diff
@@ -47,6 +144,16 @@ emit('toggle-navigation', {
 			:inert="!open || undefined"
 			@keydown.esc="handleEsc">
 			<div class="app-navigation__body" :class="{ 'app-navigation__body--no-list': !$scopedSlots.list }">
+				<NcAppNavigationSearch v-if="showSearch"
+					:label="searchLabel || undefined"
+					:value="search"
+					@update:value="$emit('update:search', $event)">
+					<template #actions>
+						<!-- @slot Optional NcAction* to show next to the search input -->
+						<slot name="search-actions" />
+					</template>
+				</NcAppNavigationSearch>
```

Comment:
Instead of adding an optional but concrete `NcAppNavigationSearch` here, can we provide a slot, so users can directly use the component, having more control on it?

This content could be the default

Reply:
The idea was to not give control there as it should always look the same, or do you mean something different?

Reply:
We are now using two different NcActions components, which means two different popovers. Don't see how it can be implemented from inline NcActionButton

Reply:
Ah I see, well then lets remove the actions and just provide the slot for optional actions?

Reply:
done

Reply:
> The idea was to not give control there as it should always look the same, or do you mean something different?

Having `NcAppNavigationSearch` exposed you still have the control but can also control `NcAppNavigationSearch`, work with its events, props, attributes.

Reply:
I am still not sure of the use case, all props and events are proxied.
And attaching any attributes make no sense as you can only attach to the wrapper element, so only `data-`, `class` or `style` would make sense.
But the idea was to not have custom styling, so having no way to assign class or style is what is intended?

Reply:
I agree here and adjusted this in the latest commit :)